site/static/examples/ingress/kong/deployment.patch.json

```diff
@@ -0,0 +1,37 @@
+{
+  "spec": {
+    "template": {
+      "spec": {
+        "containers": [
+          {
+            "name": "proxy",
+            "ports": [
+              {
+                "containerPort": 8000,
+                "hostPort": 80,
+                "name": "proxy",
+                "protocol": "TCP"
+              },
+              {
+                "containerPort": 8443,
+                "hostPort": 43,
+                "name": "proxy-ssl",
+                "protocol": "TCP"
+              }
+            ]
+          }
+        ],
+        "nodeSelector": {
+          "ingress-ready": "true"
+        },
+        "tolerations": [
+          {
+            "effect": "NoSchedule",
+            "key": "node-role.kubernetes.io/master",
+            "operator": "Equal"
+          }
+        ]
+      }
+    }
+  }
+}
```

Comment:
these json files have missing new lines.

Reply:
@neolit123 not quite sure where new lines are missing?